console/console.go

```diff
@@ -0,0 +1,103 @@
+package console
```

Comment:
This file should also have https://github.com/red-hat-storage/odf-operator/blob/main/hack/boilerplate.go.txt

Reply:
ack done.